### DISCUSSION TITLE
Start via Wireless debugging

### DISCUSSION THREAD

**Discussion by Wlz-23** [RikkaApps/Shizuku#225] [Q&A]
If I start Shizuku with via the wireless debugging feature and then disconnect from wifi, Shizuku stops working. Is this a bug or does Shizuku only work when connected to wifi cause as of right now as long as I leave my house none of the apps i have connected to Shizuku don't work and it seem counter intuitive.

**Comment by haruue** [comment]
This should be a restriction added by some device manufacturers or customized ROMs, and there is nothing Shizuku can do to bypass it.
If you are facing this issue, I'd recommend listing the device model and ROM (along with its version) you are using here.
So other users will be able to avoid purchasing those devices or flashing those ROM.

**Comment by Wlz-23** [comment]
My phone is the S22 Ultra (Snapdragon).

Will starting Shizuku with a computer have the same issues as starting with wireless debugging?

**Comment by Den4enko** [comment]
Have you enabled USB debugging in the developer settings? This is important!

**Reply by Wlz-23** [reply]
I switched over to starting Shizuku with a computer instead as it continues to run even when I disconnect.

**Comment by primesun** [comment]
I experience the issue noted above (shizuku stops to work when WiFi is turned off) with a Google pixel 6a.
